assets/tools.json

```diff
@@ -0,0 +1,3667 @@
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "NeurEco",
+        "license": "commercial",
+        "url": "https://www.adagos.com/neureco",
+        "vendor": "ADAGOS",
+        "vendorURL": "https://www.adagos.com/neureco",
+        "description": "Automatic parsimonious neural network factory",
+        "features": [],
+        "platforms": [
+            "Linux",
+            "Windows"
+        ],
+        "fmiVersions": [
+            "2.0"
+        ],
+        "fmuExport": [
+            "ME"
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "Adams",
+        "license": "commercial",
+        "url": "https://www.mscsoftware.com/Products/CAE-Tools/Adams.aspx",
+        "vendor": "MSC Software",
+        "vendorURL": "https://www.mscsoftware.com/",
+        "description": "High end multibody dynamics simulation software from MSC Software",
+        "features": [],
+        "platforms": [
+            "Linux",
+            "Windows"
+        ],
+        "fmiVersions": [
+            "1.0",
+            "2.0"
+        ],
+        "fmuExport": [
+            "CS"
+        ],
+        "fmuImport": [
+            "CS",
+            "ME"
+        ]
+    },
+    {
+        "name": "ADEPT Framework",
+        "license": "commercial",
+        "url": "https://www.adi.com/products/adept-framework/",
+        "vendor": "ADI",
+        "vendorURL": "https://www.adi.com/products/adept-framework/",
+        "description": "Industrial data and control software platform built around the concept of a data framework that links real-time Linux servers as a distributed resource and provides desktop client control of the time-deterministic computing and data handling capability",
+        "features": [],
+        "platforms": [
+            "Linux",
+            "Windows"
+        ],
+        "fmiVersions": [
+            "1.0",
+            "2.0"
+        ],
+        "fmuExport": [],
+        "fmuImport": [
+            "CS",
+            "ME"
+        ]
+    },
+    {
+        "name": "Dynamics",
+        "license": "commercial",
+        "url": "https://www.algoryx.se/products/agx-dynamics",
+        "vendor": "AGX",
+        "vendorURL": "https://www.algoryx.se/products/agx-dynamics",
+        "description": "AGX Dynamics is a professional multi-purpose physics engine for simulators, engineering, large scale granular simulators and more",
+        "features": [],
+        "platforms": [
+            "Windows"
+        ],
+        "fmiVersions": [
+            "1.0",
+            "2.0"
+        ],
+        "fmuExport": [
+            "CS"
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "Activate",
+        "license": "commercial",
+        "url": "https://solidthinking.com/product/activate",
+        "vendor": "Altair",
+        "vendorURL": "https://www.altair.com/",
+        "description": "Software environment for modeling, simulation and analysis of multi-disciplinary systems from Altair",
```

Comment:
{
        "name": "Altair Activate",
        "license": "commercial",
        "url": "https://www.altair.com/activate",
        "logo": "icon_Activate.png"
        "vendor": "Altair",
        "vendorURL": "https://www.altair.com/",
        "description": "Software environment for modeling, simulation and analysis of multi-disciplinary systems from Altair",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "1.0",
            "2.0",
            "3.0"
        ],
        "fmuExport": [
            "CS",
            "ME"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
    {
        "name": "Altair MotionSolve",
        "license": "commercial",
        "url": "https://www.altair.com/motionsolve/",
        "vendor": "Altair",
        "logo": "icon_MotionSolve.png"
        "vendorURL": "https://www.altair.com/",
        "description": "MotionSolve is a solver that performs 3D multi-body system simulations to predict the dynamic response and optimize the performance of products that move.",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "2.0"
        ],
        "fmuExport": [
            "CS"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
    {
        "name": "Altair MotionView",
        "license": "commercial",
        "url": "https://www.altair.com/motionsolve/motionview/",
        "logo": "icon_MotionView.png"
        "vendor": "Altair",
        "vendorURL": "https://www.altair.com/",
        "description": "MotionView is a graphical environment for building multibody system models and for visualizing motion simulation results using animations and plots.",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "2.0"
        ],
        "fmuExport": [
            "CS"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
![icon_Activate](https://user-images.githubusercontent.com/33484393/181022521-a5bdf2bc-41d4-4eff-895c-83b9eef4a5a2.png)
![icon_MotionSolve](https://user-images.githubusercontent.com/33484393/181022540-4261b330-5f50-4e07-8c58-588b28b12aac.png)
![icon_MotionView](https://user-images.githubusercontent.com/33484393/181022554-033ade80-23ea-4458-acff-460a7bec08e2.png)